front/app/containers/Authentication/steps/_components/TextButton.tsx

```diff
@@ -1,17 +1,9 @@
 import React from 'react';
 import styled from 'styled-components';
-import { fontSizes } from 'utils/styleUtils';
-import { darken } from 'polished'; // TODO: Remove when we have a better system
+import { ClickableTextStyles } from './TextLink';
 
 export const StyledButton = styled.button`
-  font-size: ${fontSizes.base}px;
-  color: ${({ theme }) => theme.colors.tenantText};
-  text-decoration: underline;
-
-  &:hover {
-    color: ${({ theme }) => darken(0.2, theme.colors.tenantText)};
```

Comment:
I wasn't sure whether to use this style or the one from `TextLink` (`colors.textPrimary`). So, I just picked the one that looks simpler and better in the code.